packages/hash/frontend/src/blocks/page/collab/EditorConnection.ts

```diff
@@ -202,7 +201,6 @@ export class EditorConnection {
         return this.manager.ensureDocDefined(doc).then(() => ({ doc, data }));
       })
       .then(({ data, doc }) => {
-        this.backOff = 0;
```

Comment:
What was this initially used for?

Reply:
It wasn't. It's part of the prosemirror example we started from that I just hadn't spotted wasn't being used.